ansible/roles/java/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+# Java settings
+---
+java_home: "/opt/epics-tools/lib/jvm/jdk-17"
```

Comment:
Should probably update to java 21, its already needed for 2.1.0

Reply:
Ok, noted. I will create an independent PR for that.

Reply:
Created issue #65